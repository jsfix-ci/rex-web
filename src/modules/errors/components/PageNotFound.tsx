import React from 'react';

export default () => {
  return <h1>page not found</h1>
};
