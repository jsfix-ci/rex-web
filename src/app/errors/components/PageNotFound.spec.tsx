import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from './PageNotFound';

describe('PageNotFound', () => {
  it('matches snapshot', () => {
    const tree = renderer
      .create(<PageNotFound />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
