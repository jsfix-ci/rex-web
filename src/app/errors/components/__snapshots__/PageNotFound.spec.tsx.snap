// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound matches snapshot 1`] = `
<h1>
  page not found
</h1>
`;
